src/services/file/repositories/s3.ts

```diff
@@ -254,12 +254,13 @@ export class S3FileRepository implements FileRepository {
 
       return url;
     } catch (e) {
-      // todo: usign console.log polutes the logs
-      log.error(e);
-      // todo: handle the "Not Found" error
+      if (e.name === 'NotFound') {
```

Comment:
Is there a possibility to use `instanceof`?

Reply:
I tried it first, but it does not work: it says that right hand side of `instanceof` must be a class. Not sure what they do with their code but this worked better.